src/google/google-calendars/googlecalendarsyncadaptor.h

```diff
@@ -77,7 +77,13 @@ private Q_SLOTS:
     void upsyncFinishedHandler();
 
 private:
-    QMap<int, QMap<QString, QPair<QString, QString> > > m_serverCalendarIdToSummaryAndColor;
+    struct CalendarInfo {
+        QString summary;
+        QString description;
+        QString color;
+        QString etag;
```

Comment:
CalendarInfo::etag is only assigned to, not read. I guess it is reserved for future use?